### PR TITLE
Create amended mts_transformation schema

### PR DESCRIPTION
**Pull request by dwyeradam**
### What is the context of this PR?
New schema for testing MTS transformation.

### How to review 
Matches [slides](https://docs.google.com/presentation/d/1PwbuFDxqSP89PvfzsI_KYaEVdaJOUkmTqmPnlfzv0iw/edit#slide=id.g238c536202_0_0)

### CODE REVIEW

File: data/en/mts_transformation.json [hunk 1]
```diff
@@ -0,0 +1,640 @@
...
+                                    "title": "Include",
+                                    "list": [
+                                        "VAT",
+                                        "online sales"
+                                    ]
+                                },
+                                {
+                                    "title": "Exclude",
+                                    "list": [
+                                        "sales from catering facilities to customers (e.g. in-store cafes)",
```
Comment (bitdivision) [inline data/en/mts_transformation.json:353]: 'cafes' vs 'cafés' in the slides. Not sure if issue.

File: data/en/mts_transformation.json [hunk 2]
```diff
@@ -0,0 +1,640 @@
...
+                            "currency": "GBP",
+                            "decimal_places": 2,
+                            "max_value": {
+                                "answer_id": "retail-sales-answer"
+                            },
+                            "mandatory": true
+                        }],
+                        "id": "online-sales-question",
+                        "title": "Of the <em>{{answers['retail-sales-answer']|format_currency}}</em> retail sales, how much were your <em>online sales</em>?",
+                        "description": "These are sales of goods made online, irrespective of the delivery method.",
```
Comment (bitdivision) [inline data/en/mts_transformation.json:465]: This isn't in the slides.

File: data/en/mts_transformation.json [hunk 3]
```diff
@@ -0,0 +1,640 @@
...
+                                    "q_code": "146g",
+                                    "value": "Store closures"
+                                },
+                                {
+                                    "label": "Store openings",
+                                    "q_code": "146h",
+                                    "value": "Store openings"
+                                },
+                                {
+                                    "label": "Unavailability of website",
```
Comment (bitdivision) [inline data/en/mts_transformation.json:578]: Not shown in slides.

**Review by bitdivision — COMMENTED** [review]
A few very minor content differences, not sure if these were intentional or not.

File: data/en/mts_transformation.json (hunk 3 above)
Comment (dwyeradam) [reply data/en/mts_transformation.json:578]: Yes, mistake in slides.

**Comment by jonnyshaw89** [comment]
🤷‍♂️

**Pull request closed without merging**